appconfig/appconfig.go

```diff
@@ -68,6 +89,12 @@ func Init() error {
 	var appConfig AppConfig
 	appConfig.ServerName = serverName
 
+	queryLogsWriter, err := InitQueryWriter(globalLogsWriter)
+	if err != nil {
+		return err
+	}
+	appConfig.QueryLogsWriter = &queryLogsWriter
```

Comment:
queryLogsWriter is io.Writer (interface). Interfaces are pointers by default.

Reply:
fixed